module/applications/actor/api/base-config-sheet.mjs

```diff
@@ -0,0 +1,68 @@
+import ActiveEffect5e from "../../../documents/active-effect.mjs";
+
+const { DocumentSheetV2, HandlebarsApplicationMixin } = foundry.applications.api;
+
+/**
+ * Base document sheet from which all system applications should be based.
+ */
+export default class BaseConfigSheet extends HandlebarsApplicationMixin(DocumentSheetV2) {
+  /** @override */
+  static DEFAULT_OPTIONS = {
+    classes: ["dnd5e2", "config-sheet"],
+    sheetConfig: false,
+    form: {
+      submitOnChange: true
+    }
+  };
+
+  /* -------------------------------------------- */
+  /*  Rendering                                   */
+  /* -------------------------------------------- */
+
+  /** @inheritDoc */
+  async _prepareContext(options) {
+    const context = await super._prepareContext(options);
+    context.CONFIG = CONFIG.DND5E;
+    return context;
+  }
+
+  /* -------------------------------------------- */
+  /*  Event Listeners and Handlers                */
+  /* -------------------------------------------- */
+
+  /** @inheritDoc */
+  async _onRender(context, options) {
+    await super._onRender(context, options);
+    if ( !this.isEditable ) return;
+    for ( const override of this._getActorOverrides() ) {
```

Comment:
I don't know if we really want to do this in the config sheets. If you have an AE granting you a proficiency, it should appear on your sheet in play mode. If you switch to edit mode to edit the traits, I think it should just show you the source values, never the derived ones.